tests/py/test_sync_npm.py

```diff
@@ -85,21 +83,6 @@ def test_even_deletes_package_with_linked_team(self):
         sync_npm.consume_change_stream(self.change_stream(docs), self.db)
         assert self.db.one('select * from packages') is None
 
-
-    def test_picks_up_with_last_seq(self):
```

Comment:
Gotcha. Before af41409c5bea0f3cea3afbd804e64c83ef97c693 this tested something other than our test function. Out of scope here but we could/should write a test or three for [`gratipay.cli.sync_npm.main`](https://github.com/gratipay/gratipay.com/blob/2080/gratipay/cli/sync_npm.py#L15-L40).